packages/react-core/src/components/Page/Page.tsx

```diff
@@ -72,6 +73,14 @@ export interface PageProps extends React.HTMLProps<HTMLDivElement> {
   tertiaryNav?: React.ReactNode;
   /** Accessible label, can be used to name main section */
   mainAriaLabel?: string;
+  /** Flag indicating if the tertiaryNav should be in a group */
+  isTertiaryNavGrouped?: boolean;
```

Comment:
is it a valid use case to group content but leave the tertiary nav or breadcrumbs out of it?

Reply:
This gives a user flexibility at least. I could change it to an all or nothing flag?

Reply:
okay I was curious because if it was all or nothing it could simplify the API, but after playing with the demo this is fine